modules/@ergonode/product-templates/src/store/productTemplate/actions.js

```diff
@@ -275,6 +267,7 @@ export default {
         dispatch('list/removeDisabledElement', {
             languageCode: user.language,
             elementId: layoutElements[index].id,
+            scope,
         }, {
             root: true,
         });
```

Comment:
I think that here you need to create a new method that handles the scope, and leaves this one, because it will be removed in version 1.1.0 anyway.